ckanext/stadtzhharvest/harvester.py

```diff
@@ -185,7 +195,10 @@ def gather_stage(self, harvest_job):
                     id = self._save_harvest_object(metadata, harvest_job)
                     ids.append(id)
 
-            self._create_notifications_for_deleted_datasets()
+            delete_ids = self._check_for_deleted_datasets(
```

Comment:
Do we have to call this function even though 'delete_missing_datasets' is set to false?

Reply:
Fixed.